addon/components/bs4/bs-form/element/label.js

```diff
@@ -3,7 +3,7 @@ import FormElementLabel from 'ember-bootstrap/components/base/bs-form/element/la
 export default FormElementLabel.extend({
   tagName: 'label',
 
-  classNames: ['form-control-label'],
+  classNames: ['col-form-label'],
```

Comment:
From what you told me, we should only have `col-form-label` for horizontal layouts. We still need `form-control-label` for vertical and inline layouts. The [form validation section](https://v4-alpha.getbootstrap.com/components/forms/#validation) seems inconsistent. The "How It Works" section only mentions `col-form-label`. The first example uses `form-control-label` and the second example uses `col-form-label`. Can we either check the BS4 source or ask the maintainers in an issue?

If it can be either, we should create something like a `formLabelClass` in `classNameBindings` that is a computed property returning a string for the proper class name.